src/style.css

```
@@ -1,3 +1,97 @@
+html {
+  scroll-behavior: smooth;
+}
+
+/* #region body */
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+/* #endregion */
+
+/* #region header */
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.header_logo {
+  font-size: 0;
+}
+
+/* #endregion */
+
+/* #region nav */
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+nav ul {
```

Use only class selector everywhere
```suggestion
.nav__list {
```